### PR TITLE
refactor: Meshing Workflow test.

### PR DESCRIPTION
**Pull request by prmukherj**

### CODE REVIEW

File: tests/test_new_meshing_workflow.py [hunk 1]
```diff
@@ -1054,7 +1054,20 @@ def downstream_names(task):
...
+@pytest.mark.fluent_version(">=23.2")
+def test_new_workflow_structure(new_mesh_session):
+    meshing = new_mesh_session
+    watertight = meshing.watertight()
+    assert watertight.import_geometry.arguments()
+    with pytest.raises(AttributeError) as msg:
+        watertight.TaskObject["Import Geometry"]
+    assert (
+        msg.value.args[0]
+        == "'WatertightMeshingWorkflow' object has no attribute 'TaskObject'"
```
[the remark this answers is not in the document]
Comment (prmukherj) [reply tests/test_new_meshing_workflow.py:1067]: This is a default statement provided by python. Thanks.

File: tests/test_new_meshing_workflow.py [hunk 2]
```diff
@@ -1054,7 +1054,20 @@ def downstream_names(task):
...
+    )
+
+
```
Comment (prmukherj) [inline tests/test_new_meshing_workflow.py:1070]:
```suggestion
@pytest.mark.skip("Randomly failing in CI")
```

File: tests/test_new_meshing_workflow.py [hunk 3]
```diff
@@ -1054,6 +1054,19 @@ def downstream_names(task):
...
+    meshing = new_mesh_session
+    watertight = meshing.watertight()
+    assert watertight.import_geometry.arguments()
+    with pytest.raises(AttributeError) as msg:
+        watertight.TaskObject["Import Geometry"]
+    assert (
+        msg.value.args[0]
+        == "'WatertightMeshingWorkflow' object has no attribute 'TaskObject'"
+    )
+
```
Comment (prmukherj) [inline tests/test_new_meshing_workflow.py:1069]:
```suggestion


```

File: tests/test_new_meshing_workflow.py [hunk 4]
```diff
@@ -1054,6 +1054,20 @@ def downstream_names(task):
...
+@pytest.mark.fluent_version(">=23.2")
+def test_new_workflow_structure(new_mesh_session):
+    meshing = new_mesh_session
+    watertight = meshing.watertight()
+    assert watertight.import_geometry.arguments()
+    with pytest.raises(AttributeError) as msg:
+        watertight.TaskObject["Import Geometry"]
+    assert (
+        msg.value.args[0]
+        == "'WatertightMeshingWorkflow' object has no attribute 'TaskObject'"
```
Comment (PipKat) [inline tests/test_new_meshing_workflow.py:1067]:
```suggestion
        == "'WatertightMeshingWorkflow' object has no attribute 'TaskObject'."
```
Message strings should have concluding punctuation, which is virtually always a period.

File: tests/test_new_meshing_workflow.py [hunk 5]
```diff
@@ -1055,6 +1055,20 @@ def downstream_names(task):
...
```
Comment (seanpearsonuk) [inline tests/test_new_meshing_workflow.py:1068]: don't really care about the message enough to hard-code it in the test

Comment (seanpearsonuk) [reply tests/test_new_meshing_workflow.py:1068]: As mentioned previously, `name` and `obj` attributes were added in 3.10, so when 3.10 becomes are baseline version, then we can test in terms of those, but before that point, I would prefer to leave out this kind of testing.

Comment (prmukherj) [reply tests/test_new_meshing_workflow.py:1068]: There are a lot of tests using this mechanism. I'll clean them up as well as part of a separate clean-up PR to resolve this kind of issue at once.

**Review by seanpearsonuk — APPROVED** [review]

**Review by mkundu1 — APPROVED** [review]

**Pull request merged**